sdkconfig.defaults

```diff
@@ -7,14 +25,14 @@
 #
 # Serial flasher config
 #
-#CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
-#CONFIG_ESPTOOLPY_BAUD_115200B=y
+CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
+# CONFIG_ESPTOOLPY_BAUD_115200B is not set
 # CONFIG_ESPTOOLPY_BAUD_230400B is not set
-# CONFIG_ESPTOOLPY_BAUD_921600B is not set
+CONFIG_ESPTOOLPY_BAUD_921600B=y
```

Comment:
This was what make defconfig gave me on a clean project, I rebased our makefile on that..

Apparently this was changed in the idf defaults? I think we should follow it in this case